src/db.rs

```diff
@@ -0,0 +1,67 @@
+use chrono::serde::ts_seconds_option;
+use serde::{Deserialize, Serialize};
+use sqlx::{
+    types::chrono::{DateTime, Utc},
+    FromRow, PgPool,
+};
+
+///Represents a go link, a mapping from source -> sink with some metadata
+#[derive(Debug, FromRow, Serialize, Deserialize)]
+pub struct Redirect {
+    pub source: String,
+    pub sink: String,
+    ///number of times this link has been used
+    pub usages: i32,
```

Comment:
does this need to be signed

Reply:
Postgres does not have unsigned types, so unfortunately yes